rollbar/__init__.py

```diff
@@ -702,6 +719,12 @@ def _report_exc_info(exc_info, request, extra_data, payload_data, level=None):
     if extra_trace_data and not extra_data:
         data['custom'] = extra_trace_data
 
+    # if there are feature flags attached to the exception, append that to the feature flags on
+    # singleton to create the full feature flags stack.
+    feature_flags = _feature_flags + getattr(exc_info[1], '_rollbar_feature_flags', [])[::-1]
```

Comment:
Could be that we have repeated feature flags here? I am thinking if we should use `set()` for this, but in that case we need to add a `FeatureVariation` class or similar in order to generate a unique `__hash__` since `dict` is not hashable.

Reply:
Hm. I think it's alright to have repeated feature flags here, right? Are you thinking of a case like:
```
with rollbar.watch_feature_flag('feature-a'):
  # do some stuff
  with rollbar.watch_feature_flag('feature-b'):
    # do some other stuff
    with rollbar.watch_feature_flag('feature-a'):
      # do some more stuff
```